packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -628,6 +634,11 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         weth_ = WETH;
     }
 
+    /// @notice Returns the chain ID of the L2 network this contract argues about.
+    function chainId() external view returns (uint256 chainId_) {
```

Comment:
nit: For clarity should we call this `l2ChainId()` and the immutable `L2_CHAIN_ID`?

Reply:
Good catch